src/Tribe/Timezones.php

```diff
@@ -615,7 +615,7 @@ public static function get_valid_timezone( $timezone_candidate ) {
 			return $timezone_candidate->getName();
 		}
 
-		$timezone_string = preg_replace( '/(\\+||\\-)0$/', '', $timezone_candidate );
+		$timezone_string = preg_replace( '/[+\-]0$/', '', $timezone_candidate );
```

Comment:
```suggestion
		$timezone_string = preg_replace( '/[+-]0$/', '', $timezone_candidate );
```

Dash at end of range doesn't have to be escaped: https://stackoverflow.com/questions/30398445/matching-dot-in-a-range-preg-match